airframe-sql/src/test/scala/wvlet/airframe/sql/analyzer/TypeResolverTest.scala

```diff
@@ -1070,4 +1070,19 @@ class TypeResolverTest extends AirSpec with ResolverTestHelper {
     analyze("select \"prénom\" from (select name as \"prénom\" from A)")
     // No error
   }
+
+  test("resolve CTE in AliasedRelation") {
+    val p1 = analyze("with t1 as (select id from A) select id from (select id from t1) t2")
+    p1.outputAttributes shouldMatch { case List(col: ResolvedAttribute) =>
+      col.fullName shouldBe "id"
```

Comment:
I'm not sure if adding a qualifier to columns only when column is referred inside function call is expected.

Reply:
In my current understanding, qualifier needs to be added only when the user adds it in the input SQL. If the column cannot be properly resolved due to lack of the qualifier not given in the SQL, there should be other types of bugs in TypeResolver side.